src/main/java/usecase_essence_use/data_calculator/CalculatorCollectible.java

```diff
@@ -1,13 +1,11 @@
-package usecase_heal_and_upgrade;
+package usecase_essence_use.data_calculator;
 
 public class CalculatorCollectible {
-    int healEssenceRate;
-    int upgradeEssenceNeed;
 
-    public CalculatorCollectible(){
-        healEssenceRate = 1;
-        upgradeEssenceNeed =20;
-    }
+    public CalculatorCollectible(){}
+
+    public final int HP_ESSENCE_EXCHANGE_RATE = 1;
+    public final int ESSENCE_FOR_UPGRADE = 20;
```

Comment:
Naming convention is incorrect,
https://docs.oracle.com/javase/tutorial/java/nutsandbolts/variables.html
should just be hpEssenceExchangeRate and essenceForUpgrade

Reply:
fixed